tests/framework/ensembleModelTests/test_ensemble_model_2_codes_optional_output.xml

```diff
@@ -6,14 +6,15 @@
     <created>2017-05-17</created>
     <classesTested>Models.Code, JobHandler.Thread</classesTested>
     <description>
-       The main goal of this test is to show how to specify optional output in the Ensemle Model object.
+       The main goal of this test is to show how to specify optional output in the Ensemble Model object.
+       All the RAVEN available data storage types are tested (DataObjects and Databases).
     </description>
```

Comment:
***Maybe we could have a git hook that detects changes to xml files and asks a developer to write a brief description at commit time for the specific test file(s) edited?***

Good Idea...